app/src/androidTest/java/com/project/symptoms/db/dao/GlucoseDaoImplTest.java

```diff
@@ -0,0 +1,120 @@
+package com.project.symptoms.db.dao;
+
```

Comment:
Why the test class is not in test directory?

Reply:
because this test runs using the real app context